torch/_torch_docs.py

```diff
@@ -3335,7 +3335,7 @@ def merge_dicts(*dicts):
 Args:
     {input}
     dim (int): the dimension in which we index
-    index (LongTensor): the 1-D tensor containing the indices to index
+    index (IntTensor or LongTensor): the 1-D tensor containing the indices to index
```

Comment:
I don't think IntTensor is supported on cuda?

Reply:
index_add and index_select on cuda are enabled in this PR. Github folds long files. Need to avoid adding more to this PR:)